tests/mocked/test_databases.py

```diff
@@ -712,7 +711,8 @@ def test_databases_list_users(mock_client: Client, mock_client_url):
     resp = mock_client.databases.list_users(cluster_uuid)
 
     assert expected == resp
-=======
+
+
 def test_databases_delete(mock_client: Client, mock_client_url):
```

Comment:
I'm not able fix this test ! @danaelhe if you could help me with this

Reply:
Easy fix! You'll just need to add `@responses.activate` right before the function:
```suggestion
@responses.activate
def test_databases_delete(mock_client: Client, mock_client_url):
```